flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/dialect/JDBCDialects.java

```diff
@@ -407,7 +407,6 @@ public int minTimestampPrecision() {
 			//  PrimitiveArrayTypeInfo.BYTE_PRIMITIVE_ARRAY_TYPE_INFO in LegacyTypeInfoDataTypeConverter.
 			return Arrays.asList(
 					LogicalTypeRoot.BINARY,
-					LogicalTypeRoot.TIMESTAMP_WITH_LOCAL_TIME_ZONE,
```

Comment:
If we don't support `TIMESTAMP_WITH_LOCAL_TIME_ZONE` for now, why remove it from the unsupported list?

Reply:
it's orthogonal. PG and Flink's pg catalog/source/sink both support timestamp with local time zone, however it cannot be used due to precision conversion between type systems. once the conversion issue is resolved, it should be good to go